lms/djangoapps/courseware/tests/helpers.py

```diff
@@ -72,8 +72,8 @@ def new_module_runtime(self, **kwargs):
         """
         return get_test_system(course_id=self.course.id, **kwargs)
 
-    def new_descriptor_runtime(self, **kwargs):
-        runtime = get_test_descriptor_system(**kwargs)
+    def new_block_runtime(self, **kwargs):
```

Comment:
This one is still relevant. Let's revert it.

Reply:
reverted